src/python/build_management/build_manager.py

```diff
@@ -1111,84 +1151,88 @@ def get_revisions_list(bucket_path, testcase=None):
   return revision_list
 
 
-def setup_trunk_build():
+def setup_trunk_build(bucket_paths_env_vars=DEFAULT_BUILD_BUCKET_PATH_ENV_VARS,
+                      build_prefix=None):
   """Sets up latest trunk build."""
-  release_build_bucket_path = environment.get_value('RELEASE_BUILD_BUCKET_PATH')
-  sym_release_build_bucket_path = environment.get_value(
-      'SYM_RELEASE_BUILD_BUCKET_PATH')
-  sym_debug_build_bucket_path = environment.get_value(
-      'SYM_DEBUG_BUILD_BUCKET_PATH')
+  if not bucket_paths_env_vars:
+    return None
 
-  base_build_dir = _base_build_dir(release_build_bucket_path)
-  _setup_build_directories(base_build_dir)
+  build_urls = []
+  for env_var in bucket_paths_env_vars:
+    bucket_path = environment.get_value(env_var)
+    if not bucket_path:
+      logs.log_warn('Build bucket path is not specified as %s.' % env_var)
+      continue
 
-  release_build_urls = get_build_urls_list(release_build_bucket_path)
-  if not release_build_urls:
-    logs.log_error('Error getting list of release build urls from %s.' %
-                   release_build_bucket_path)
+    urls_list = get_build_urls_list(bucket_path)
+    if not urls_list:
+      logs.log_error('Error getting list of build urls from %s.' % bucket_path)
+      return None
+
+    build_urls.append({'bucket_path': bucket_path, 'urls_list': urls_list})
```

Comment:
nit: use a NamedTuple for these instead of a dict for each entry.

Reply:
Done!